tools/AduCmdlets/create-adu-import-manifest.sh

```diff
@@ -221,7 +221,8 @@ for file in "${UPDATE_FILES[@]}"; do
     fi
 done
 
-CREATED_DATETIME=$(date --utc --iso-8601=seconds)
+# ISO 8601 UTC date time with Z designator.
+CREATED_DATETIME=$(date --utc +%FT%TZ)
```

Comment:
i'd prefer that we not change this script and we wait until the service is fixed.

changing this script will be confusing as it looks like we DONT support ISO8601, when we will (and should!)

Reply:
There is *no* rush to get this working AFAIK.

Reply:
This is temporary. We will revert this next week once server side fix is fully deployed. ETA Thursday.